azure_functions_worker/loader.py

```diff
@@ -225,7 +227,9 @@ def index_function_app(function_path: str):
                     f"level function app instances are defined.")
 
     if not app:
+        script_file_name = get_app_setting(setting=SCRIPT_FILE_NAME,
```

Comment:
This should be tested in an existing test - else we should cover this - as this is a good case - function_app.py without `app` in it.

Reply:
Didn't see this being tested anywhere else, so added this as a new test case for invalid V2 apps.